scripts/devel/create_devel_task_generator.py

```diff
@@ -73,17 +73,18 @@ def main(argv=sys.argv[1:]):
              'and instead of installing the tests are ran')
     args = parser.parse_args(argv)
 
+    condition_context = dict(os.environ)
```

Comment:
I think you actually want to use the build environment variables argument rather than the environment this script will be run in.

This is how the build arguments are passed through to the running container. (Introduced in #554)

```suggestion
    condition_context = args.env_vars
```

Reply:
Good point. Fixed in 9c5e875ba72b3531b385428f6ae9c0b2a02f5c58.

`args.env_vars` is a list of strings so it needs to be converted into a `dict`.